src/pipx/main.py

```diff
@@ -183,6 +183,18 @@ def package_is_url(package: str, raise_error: bool = True) -> bool:
     return False
 
 
+def package_is_abspath(package: str):
```

Comment:
Perhaps we should change the function name as well? Since this check can be applied to both absolute and relative paths.


```suggestion
def package_is_path(package: str):
```

Reply:
Good catch, I have changed all the reference and changelog accordingly.